src/pages/Dashboard/UsageTiles/CycleUsage-Tile.vue

```diff
@@ -28,7 +28,7 @@ export default {
     freeUsage() {
       if (isNaN(this.usage)) return null
       const percentage = this.usage / 10000
-      return percentage > 1 ? 100 : percentage * 100
+      return percentage > 1 ? 100 : (percentage * 100).toFixed(2)
```

Comment:
I wonder if we need the decimal at all, maybe we can use `.toFixed()` instead

Reply:
I'm fine with that!